lib/foreman_ansible/register.rb

```diff
@@ -96,6 +96,8 @@
   parameter_filter Host::Managed, base_role_assignment_params.merge(:host_ansible_roles_attributes => {})
   parameter_filter Hostgroup, base_role_assignment_params.merge(:hostgroup_ansible_roles_attributes => {})
 
+  register_global_js_file 'fills'
```

Comment:
We use a different pattern, instead of having a global file just for fills, we have a global index file for any global need (i.e routes registrations).

For further information and some examples, please have a look at the plugin template: https://github.com/theforeman/foreman_plugin_template/blob/master/webpack/global_index.js